src/pages/index.vue

```diff
@@ -0,0 +1,212 @@
+        >
+          Cancel
+        </v-btn>
+        <v-btn
+          class="text-none flex-grow-1 mr-4"
+          :disabled="!!!newItem.title"
+          :color="newItem.edit ? 'orange-accent-3' : 'green-accent-4'"
+          :prepend-icon="newItem.edit ? 'mdi-check' : 'mdi-plus'"
+          variant="flat"
+          size="large"
+        >
+          {{ newItem.edit ? "Update" : "Add" }}
+        </v-btn>
+        <v-switch
+          color="success"
+          v-model="isEdit"
+          label="Edit mode"
+          hide-details
+          inset
+        ></v-switch>
+      </div>
+    </v-col>
+  </v-row>
+  <!-- list -->
+  <v-list lines="one">
+    <v-list-item v-for="item in filteredList" :key="item.id" :value="item.id">
+      <template #prepend="{}">
+        <v-list-item-action start class="list-item">
+          <v-checkbox-btn v-model="item.isActive"></v-checkbox-btn>
+        </v-list-item-action>
+      </template>
+      <v-list-item-title>{{ item.title }}</v-list-item-title>
+      <v-list-tile-sub-title>
+        <v-chip class="mr-2" color="pink" label>
+          <v-icon icon="mdi-shape" start></v-icon>
+          {{brands.find((brand) => brand.id === item.brandId).title}}
+        </v-chip>
+        <v-chip v-for="tag in item.tags" :key="tag" class="mr-2" color="primary">
+          {{ tag }}
+        </v-chip>
+      </v-list-tile-sub-title>
+      <template v-slot:append="{ isActive }">
+        <v-list-item-action v-if="isEdit" end>
+          <v-col cols="auto">
+            <v-btn
+              color="orange-accent-3"
+              icon="mdi-pencil"
+              @click.stop="onEditEnable(item)"
+            ></v-btn>
+          </v-col>
+          <v-col cols="auto">
+            <v-btn
+              variant="tonal"
+              color="red-accent-3"
+              icon="mdi-delete"
+              @click.stop="onDeleteConfirmDialog(item)"
+            ></v-btn>
+          </v-col>
+        </v-list-item-action>
+      </template>
+    </v-list-item>
+  </v-list>
+  <!-- delete model -->
+  <v-dialog v-model="deleteItem" width="auto">
+    <v-card>
+      <v-card-title class="text-h5"> Are you sure? </v-card-title>
+      <v-card-text>Once deleted you cannot recover</v-card-text>
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="green-darken-1" variant="text" @click="deleteItem = undefined">
+          Cancel
+        </v-btn>
+        <v-btn color="red-darken-1" variant="text" @click="onDeleteConfirm(item)">
+          Yes, delete
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts" setup>
+import { useListStore } from "@/stores/list";
+
+const store = useListStore();
+
+const { list, brands, categories } = store;
+
+const newItem = ref({
+  title: "",
+  brandId: null,
+  edit: false,
+  isActive: false,
+  categories: [],
+});
+
+const filteredList = computed(() =>
+  // eslint-disable-next-line no-extra-boolean-cast
```

Comment:
What is the reason for disabling eslint? 
Is it necessary?

Reply:
!!! (triple negation) shows eslint warnings but it's a shorthand for checking not empty strings